src/sphinx/formats/universal.rst

```diff
@@ -351,46 +360,91 @@ You get a set of methods which will help you to create mappings very easily.
 Mapping Examples
 ~~~~~~~~~~~~~~~~
 
-SBT provides and IO and `Path`_ API, which
-lets you define custom mappings easily. The files will appear in the generate universal zip, but also in your
+SBT provides the `IO`_ and `Path`_ APIs, which
+help make defining custom mappings easy. The files will appear in the generate universal zip, but also in your
 debian/rpm/msi/dmg builds as described above in the conventions.
 
+.. _IO: http://www.scala-sbt.org/0.13.1/docs/Detailed-Topics/Paths.html
 .. _Path: http://www.scala-sbt.org/0.13.1/docs/Detailed-Topics/Paths.html
 
-The ``packageBin in Compile`` dependency is only needed, if your files get generated
+The ``packageBin in Compile`` dependency is only needed if your files get generated
 during the ``packageBin`` command or before. For static files you can remove it.
 
 Mapping a complete directory
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+There are some helper methods so you can create a mapping for a complete directory:
+
+For static content, you can just add the directory to the mapping:
+
+.. code-block:: scala
+
+    mappings in Universal ++= directory("SomeDirectoryNameToInclude")
+
+If you want to add everything in a directory where the path for the directory is dynamic, e.g. the ``scala-2.10/api`` directory that is nested under in the ``target`` directory, and ``target`` is defined in a task:
+
+.. code-block:: scala
+
+    mappings in Universal <++= (packageBin in Compile, target) map { (_, target) =>
+      directory(target / "scala-2.10" / "api")
+    }
```

Comment:
Will do.  
 I think this is one of the examples that would be great in the intro / main concepts section.  (For a later PR)